bin/sandbox

```diff
@@ -3,6 +3,7 @@
 # Used in the sandbox rake task in Rakefile
 
 set -e
+test -z "${DEBUG+empty_string}" && set -x
```

Comment:
Careful! Length is zero (`-z`) when `$DEBUG is not set, so it's actually doing the opposite that we want.

Reply:
🤦‍♂️ thanks, fixed